### PR TITLE
Feature/integrable

### PR DESCRIPTION
**Pull request by AdrienHorgnies**
Now we will actually test if our software does something

This isn't a finished product but now we're able to know if we messed up along the way.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -71,7 +72,12 @@ const openSession = credentials => db.connect(credentials)
  * @resolves {selectEntities({driver, connection, schema, results: {tables, twoTypeJunction, jhipster, liquibase}})} session
  */
 const getEntityCandidates = session => db.entityCandidates(session)
-    .then(session => selectEntities(session)); // store all the tables of the database we're connected to into the session
+    .then((session) => {
+        if (session.mode === cst.modes.manual) {
+            return selectEntities(session);
+        }
+        return setEntities(session);
+    }); // store all the tables of the database we're connected to into the session
```
Comment (bastienmichaux) [inline index.js:81]: That comment is out of place

File: lib/default-choices.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const entities = session => Object.assign({ entities: session.results.tables }, session);
```
Comment (bastienmichaux) [inline lib/default-choices.js:10]: pls document why that function exists, so we know why it's here

File: index.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /**
  * @file Entry point of the program
  *
- * Uses a chain of function, meaning each function starts the next function
+ * Uses a chained functions, meaning each function calls the next function
```
Comment (bastienmichaux) [inline index.js:4]: delete the 'a' (captain nitpicking here)

File: lib/exportEntities.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+const fse = require('fs-extra');
+const cst = require('../constants');
+
+const exportEntities = entities => fse.writeJson(cst.exportFile, entities, { spaces: 2 });
```
Comment (bastienmichaux) [inline lib/exportEntities.js]: could you use 'spaces: 4' ? That's the preferred format for Jhipster and most web apps as far as I know.

File: travis/templates/mysql/dev-life-export.json [hunk 5]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (bastienmichaux) [inline travis/templates/mysql/dev-life-export.json:1]: Since I'm nitpicking about the JSON indentation, don't forget to change the indentation here too

**Review by bastienmichaux — CHANGES_REQUESTED** [review]

File: lib/exportEntities.js (hunk 4 above)
Comment (AdrienHorgnies) [reply lib/exportEntities.js]: Will do

File: travis/templates/mysql/dev-life-export.json (hunk 5 above)
Comment (AdrienHorgnies) [reply travis/templates/mysql/dev-life-export.json:1]: That's the same indentation, I must change both or the build will fail.

**Pull request merged**